src/components/calcite-block/calcite-block.scss

```diff
@@ -79,3 +83,16 @@ calcite-loader[inline] {
   padding: 0 var(--calcite-app-side-spacing-three-quarters) var(--calcite-app-cap-spacing-half);
   position: relative;
 }
+
+::slotted([slot="control"]) {
+  position: absolute;
```

Comment:
Could the control be positioned flex instead of absolute over the header?

Reply:
That messes up the header layout.

Reply:
Sorry, I should've added more info. I tried something similar, but I need the control to be placed above the header toggle. Otherwise, it looks weird when focused:

<img width="1064" alt="Screen Shot 2020-01-16 at 9 01 05 AM" src="https://user-images.githubusercontent.com/197440/72546076-f94e2f80-383e-11ea-91b8-267b3b543400.png">